library/swift/FinalStreamIntel.swift

```diff
@@ -2,30 +2,31 @@
 import Foundation
 
 /// Exposes one time HTTP stream metrics, context, and other details.
+/// Note: -1 means "not present" for the fields of type Int64.
```

Comment:
Why wouldn't we use an `Optional` here, i.e. the language's built-in mechanism to indicate that a value is "not present"? That would prevent some classes of misuse errors for these fields.

Reply:
Well, this PR goal does not really change the situation. In reality, there was an important comment that was missing previously:
`/// Note: 0 means "not present" for any field where its name ends with Ms`
And I agree that this is a suboptimal API. But it is on par with the Kotlin and Java versions - same pattern.
My guess is dedicated PR for all 3 languages would be appropriate.